curriculum/challenges/english/01-responsive-web-design/applied-accessibility/standardize-times-with-the-html5-datetime-attribute.english.md

```diff
@@ -23,13 +23,13 @@ Camper Cat's Mortal Kombat survey results are in! Wrap a <code>time</code> tag a
 ```yml
 tests:
   - text: Your <code>time</code> tags should wrap around the text "Thursday, September 15&lt;sup&gt;th&lt;/sup&gt;".
-    testString: assert($('time').text().match(/Thursday, September 15th/g), 'Your <code>time</code> tags should wrap around the text "Thursday, September 15&lt;sup&gt;th&lt;/sup&gt;".');
+    testString: assert($('time').html().trim() === "Thursday, September 15<sup>th</sup>");
   - text: Your <code>time</code> tag should have a <code>datetime</code> attribute that is not empty.
-    testString: assert($('time').attr('datetime'), 'Your <code>time</code> tag should have a <code>datetime</code> attribute that is not empty.');
+    testString: assert($('time').attr('datetime'));
```

Comment:
This test has always been a problem for me.  It shows passing without any modifications to the code, because there are other `time` elements with `datetime` attributes already.  I suggest the following change to fix this:
```suggestion
    testString: assert($('time[datetime]').length === 4);
```

Reply:
Ok I made the changes. It works but the only thing i noticed is when the first <code>datetime</code> attr is left empty the challenge still passes for that individual test but not the group of tests. This is due to the other test checking if its equal to the specific date . <code>Your datetime attribute should be set to a value of 2016-09-15.</code>

can I try to fix it so that the individual test won't pass if its empty? or is it good enough?

Reply:
it works in the sense that it won't pass right away anymore but it will pass if there is an empty <code>datetime</code> attribute

Reply:
Which test?

Reply:
take a look at this code if we do 
<code>assert($('time[datetime]').length === 4 && $('time').attr('datetime') !== "");</code>
the code then works perfectly as it should!  let me know what you think

Reply:
the test is 2nd one on Applied Accessibility: Standardize Times with the HTML5 datetime Attribute :
Your <code>time</code> tag should have a <code>datetime</code> attribute that is not empty.